teuthology/task/internal.py

```diff
@@ -359,6 +359,43 @@ def check_conflict(ctx, config):
     if failed:
         raise RuntimeError('Stale jobs detected, aborting.')
 
+def check_coredumps(path, remote):
```

Comment:
The name of this function doesn't really describe what it does. Maybe ``fetch_binaries_for_coredumps()`` ?